15-eco.Rmd

```diff
@@ -25,44 +25,37 @@ library(vegan)             # community ecology package
 
 ## Introduction
 
-In this chapter we will model the floristic gradient of fog oases to reveal distinctive vegetation belts that are clearly controlled by water availability.
-To do so, we will bring together concepts presented in previous chapters and even extend them (Chapters \@ref(spatial-class) to \@ref(geometry-operations) and  Chapters \@ref(gis) and \@ref(spatial-cv)).
+This chapter models the floristic gradient of fog oases to reveal distinctive vegetation belts that are clearly controlled by water availability.
+The example, building on previous applied chapters, provides an opportunity to bring together and extend concepts presented in previous, to further enhance your skills at using R for geocomputation.
```

Comment:
There is something amiss with this sentence.

Reply:
True that. Now reads:

> The case study provides an opportunity to bring together and extend concepts presented in previous, chapters to further enhance your skills at using R for geocomputation.